src/components/events-list/index.js

```diff
@@ -45,7 +45,8 @@ export const EventsList = ({ events }) => (
 );
 
 EventsList.propTypes = {
-  events: PropTypes.arrayOf(PropTypes.shape(eventModel)).isRequired,
+  events: PropTypes.arrayOf(PropTypes.shape(EventModelValidationMap))
+    .isRequired,
 };
 
 export default EventsList;
```

Comment:
The default export is not used anywhere and can be removed

Reply:
Thanks for the observation. I will have this fixed